components/script_traits/lib.rs

```diff
@@ -163,6 +163,11 @@ pub struct LoadData {
     pub referrer: Option<Referrer>,
     /// The referrer policy.
     pub referrer_policy: Option<ReferrerPolicy>,
+
+    /// add a field to store srcdoc contents - init step 3; since srcdoc is a DOMString which is
+    /// inherently a Rust String - https://doc.servo.org/script/dom/bindings/str/struct.DOMString.html
+    /// https://doc.rust-lang.org/rust-by-example/std/str.html#targetText=Strings,in%20Rust%3A%20String%20and%20%26str%20.&targetText=%26str%20is%20a%20slice%20(%20%26%5B,view%20into%20Vec%20.
```

Comment:
Let's replace this comment with `/// The source to use instead of a network response for a srcdoc document.`